quality_control_stock_oca/models/qc_inspection.py

```diff
@@ -47,13 +39,20 @@ def _compute_lot(self):
             if inspection.object_id._name == "stock.move":
                 inspection.lot_id = first(
                     move_lines.filtered(
-                        lambda x, inspection=inspection: x.move_id
-                        == inspection.object_id
+                        lambda x, ins=inspection: x.move_id == ins.object_id
                     )
                 ).lot_id
             elif inspection.object_id._name == "stock.lot":
                 inspection.lot_id = inspection.object_id
 
+    @api.depends("object_id")
```

Comment:
REstore at the same point in order to avoid extra changes please

Reply:
Done